tests/inputFields.spec.ts

```diff
@@ -22,22 +22,22 @@ test.describe('Input fields', () => {
         const updateButton = page.getByRole('button', {name: "Update"})
         await updateButton.click()
         //Asserting the value of the first row is updated
-        await expect(tableRows.locator('[id="0"]')).toHaveValue('rabbit')
+        await expect(tableRow.locator('[id="0"]')).toHaveValue('rabbit')
         //Now going to edit the same row again to back 'cat'
-        await tableRows.getByRole('button',{name: "Edit"}).first().click()
+        await tableRow.getByRole('button',{name: "Edit"}).first().click()
         //Clearing 'rabbit' from the field and filling with 'cat' and clicking 'Update' button
         await nameInputField.click()
         await nameInputField.clear()
         await nameInputField.fill('cat')
         await updateButton.click()
         //Then asserting the 'cat' value of the same row is updated
-        await expect(tableRows.locator('[id="0"]')).toHaveValue('cat')
+        await expect(tableRow.locator('[id="0"]')).toHaveValue('cat')
     })
 
     test('Test Case 2: Cancel pet type update', async ({page}) => {
-        //Creating locator for targeted table rows that is going to be edited
-        const tableRows = page.getByRole('row', { name: 'dog' })
-        await tableRows.getByRole('button', {name: "Edit"}).click()
+        //Creating locator for targeted table row that is going to be edited
+        const tableRow = page.getByRole('row', { name: 'dog' })
```

Comment:
When picking the names, try to be as specific as possible, it will not hurt.
So this locator is not just some table row. It is a `dogTableRow` or `tableRowOfDogType`, something like that.

It may seem like a not big deal to you right now, but trust me, descriptive naming helps a lot later, when you read more complex code which you haven't see for one month and forgot what was it about.

Reply:
It's no longer a dynamic value. This constant points to the row with a "dog." It can't resolve anything else, "cat" row for example. If the "dog" value is not there, the locator will just fail.